tests/Tests/400__ods_dashboard/adminUITest.robot

```diff
@@ -0,0 +1,159 @@
+*** Settings ***
+Library         OpenShiftCLI
+Library         OpenShiftLibrary
+Resource        ../../Resources/Common.robot
+Resource        ../../Resources/Page/ODH/JupyterHub/JupyterHubSpawner.robot
+
+
+*** Test Cases ***
+Verify If Unauthorisez User Can Not Change The Permission
+    [Tags]  ODS-1661
+    ...     ODS-1660
+    Set Library Search Order  SeleniumLibrary
+    Check User Management Option Is Available For The User   ${TEST_USER.USERNAME}   ${TEST_USER.PASSWORD}  ${TEST_USER.AUTH_TYPE}
+    Clear All The Input Values In User Managermant
+    Add OpenShift Groups To Data Science Administrators         cluster-admins  rhods-admins
+    Add OpenShift Users groups to Data Science administrators   system:authenticated
+    Save Changes In User Management Setting
+
+    Verify Expect Changes In AdminGroups Are Present In CRD     cluster-admins  rhods-admins
+    Verify Expect Changes In AllowedGroups Are Present In CRD   system:authenticated
+
+    Check User Management Option Is Available For The User     ${OCP_ADMIN_USER.USERNAME}   ${OCP_ADMIN_USER.PASSWORD}  ${OCP_ADMIN_USER.AUTH_TYPE}
+    Remove OpenShift Groups To Data Science Administrators     rhods-admins
+    Save Changes In User Management Setting
+    Switch Browser  1
+    Add OpenShift Groups To Data Science Administrators         rhods-noaccess
+    Add OpenShift Users Groups To Data Science Administrators   rhods-users
+
+    Verify Expect Changes In AdminGroups Are Present In CRD     cluster-admins
+    Verify Expect Changes In AllowedGroups Are Present In CRD   system:authenticated
```

Comment:
the previous lines (29,30) added other groups, so you may need to check those are present in CRD too

Reply:
nope     CRD should not contain rhods-noaccess , rhods-users
as in we have removed  `rhods-admins` earlier

Reply:
hm so why adding them in the previous 2 lines (26-27 now). Are you just checking that if you add some groups but you don't save options then they shouldn't be set in the CRD?

Reply:
as discussed on the call hope its clear :)